README.md

```diff
@@ -69,23 +62,23 @@ NumPy. This is optional, but using an optimized BLAS such as [ATLAS] or
 magnitude. On OS X, NumPy picks up the BLAS that comes with it
 automatically, so you don’t need to do anything special.
 
-The simple way to install gensim is:
+Install the latest version of gensim:
 
-    pip install -U gensim
+```bash
+    pip install --upgrade gensim
+```
 
 Or, if you have instead downloaded and unzipped the [source tar.gz]
-package, you’d run:
+package:
 
-    python setup.py test
+```bash
     python setup.py install
+```
 
-For alternative modes of installation (without root privileges,
-development installation, optional install features), see the
-[documentation].
+For alternative modes of installation, see the [documentation].
 
-This version has been tested under Python 2.7, 3.5 and 3.6. Gensim’s github repo is hooked
-against [Travis CI for automated testing] on every commit push and pull
-request. Support for Python 2.6, 3.3 and 3.4 was dropped in gensim 1.0.0. Install gensim 0.13.4 if you *must* use Python 2.6, 3.3 or 3.4. Support for Python 2.5 was dropped in gensim 0.10.0; install gensim 0.9.1 if you *must* use Python 2.5). 
+Gensim is being [continuously tested](https://travis-ci.org/RaRe-Technologies/gensim) under Python 3.5, 3.6, 3.7 and 3.8.
```

Comment:
`+ appveyor here`, `+ gensim-wheels here`

Reply:
Note to self: remove 3.5, we don't actually test that. https://github.com/RaRe-Technologies/gensim/pull/2814#issuecomment-621562196